### PR TITLE
Update exchange protocol documentation

### PR DESCRIPTION
**Pull request by arhimondr**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/develop/worker-protocol.rst [hunk 1]
```diff
@@ -84,12 +85,16 @@ response includes:
...
-the next chunk of results. The client keeps fetching results until it receives
-``X-Presto-Buffer-Complete`` HTTP header with the value of "true".
+After receiving the first chunk of results, the client uses the
+``X-Presto-Page-End-Sequence-Id`` sequence number to request the next chunk of results.
+Requesting the next chunk automatically acknowledges reception of the previous chunk.
+The client keeps fetching results until it receives ``X-Presto-Buffer-Complete`` HTTP header
+with the value of "true".
+
+When a client decides not to fetch the next chunk of data right away it sends an
+explicit ack via a GET on ``{taskId}/results/{bufferId}/{token}/acknowledge``. The client
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/develop/worker-protocol.rst:95]:
```suggestion
explicit ack using a GET on ``{taskId}/results/{bufferId}/{token}/acknowledge``. The client
```
Do not use Latin abbreviations. See the [Gitlab documentation style guide recommended word list entry for via](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#via) for discussion and suggestions. 

I chose one of the suggestions to make here, but if you feel one of the others in the linked entry for via fits better, please change my suggested _using_ to something else!

**Review by steveburnett — CHANGES_REQUESTED** [review]
Thanks for the docs! Two nits, nothing else.

File: presto-docs/src/main/sphinx/develop/worker-protocol.rst (hunk 1 above)
Comment (elharo) [reply presto-docs/src/main/sphinx/develop/worker-protocol.rst:95]: Putting on my Latin major hat, Gitlab is wrong about this one. "via" is a perfectly fine English word and not a Latin abbreviation. Latin abbreviations to be avoided include i.e., e.g., and n.b. but not "via"

Via is also a Latin word that means something different (road) than it means in English.

https://www.merriam-webster.com/dictionary/via

File: presto-docs/src/main/sphinx/develop/worker-protocol.rst [hunk 2]
```diff
@@ -84,12 +85,16 @@ response includes:
...
-along with the results. Then, the client uses that sequence number to request
-the next chunk of results. The client keeps fetching results until it receives
-``X-Presto-Buffer-Complete`` HTTP header with the value of "true".
+After receiving the first chunk of results, the client uses the
+``X-Presto-Page-End-Sequence-Id`` sequence number to request the next chunk of results.
+Requesting the next chunk automatically acknowledges reception of the previous chunk.
+The client keeps fetching results until it receives ``X-Presto-Buffer-Complete`` HTTP header
+with the value of "true".
+
+When a client decides not to fetch the next chunk of data right away it sends an
```
Comment (elharo) [inline presto-docs/src/main/sphinx/develop/worker-protocol.rst:94]: comma after away

File: presto-docs/src/main/sphinx/develop/worker-protocol.rst (hunk 1 above)
Comment (steveburnett) [reply presto-docs/src/main/sphinx/develop/worker-protocol.rst:95]: True: it's not an _abbreviation_, and I'm aware of the meaning of the Latin word, but the use of via in technical documentation is recommended against as a standard practice. In addition to Gitlab, the [Google documentation style guide also recommends against use of via](https://developers.google.com/style/word-list#via). 

Does that make sense?

**Comment by arhimondr** [comment]
Thank you for the review @steveburnett, @elharo . Updated.

**Review by steveburnett — APPROVED** [review]
LGTM! (docs) 

Pull updated branch, new local doc build, looks good. Thanks!

**Pull request merged**